src/cwv/handler.js

```diff
@@ -27,9 +42,18 @@ export async function CWVRunner(auditUrl, context, site) {
     interval: INTERVAL,
     granularity: 'hourly',
   };
-  const cwvData = await rumAPIClient.query('cwv', options);
+  const cwvData = await rumAPIClient.queryMulti(CWV_QUERIES, options);
   const auditResult = {
-    cwv: cwvData.filter((data) => data.pageviews >= DAILY_THRESHOLD * INTERVAL),
+    cwv: cwvData.cwv.filter((data) => data.pageviews >= DAILY_THRESHOLD * INTERVAL)
+      .map((cwvItem) => {
+        // Find a matching formVital by URL
+        const matchingFormVital = cwvData['form-vitals'].find(
+          (formVital) => formVital.url === cwvItem.url,
+        );
+
+        const hasForm = matchingFormVital ? checkHasForm(matchingFormVital) : false;
```

Comment:
IIUC, formVitals give you only urls which have formsubmit/formview/formengagement ? If that is true, do you think we need this check

Reply:
yep, agreed. Removed it.